### PR TITLE
update release note files

### PR DESCRIPTION
**Pull request by LisaDelaney**

### CODE REVIEW

**Comment by LisaDelaney** [comment]
@samjwu to rebuild CHANGELOG with these updated files.

File: tools/autotag/templates/rocm_changes/5.0.0.md [hunk 1]
```diff
@@ -301,38 +306,44 @@ The correct order is:
...
+  counter(s) to monitor, and save this to a different output file using the -o flag.
 
 3. Check the output result file from step 1.
 
-4. The order of timestamps correctly displays as:
-  DispatchNS < BeginNS < EndNS < CompleteNS
+4. The order of timestamps correctly displays as: DispatchNS < BeginNS < EndNS < CompleteNS
 
-5. Users can find the values of the collected counters in the output file generated in step 2.
+1. Users can find the values of the collected counters in the output file generated in step 2.
```
Comment (Rmalavally) [inline tools/autotag/templates/rocm_changes/5.0.0.md:326]: We are not restarting the sequence of steps. Perhaps a note?

Comment (LisaDelaney) [reply tools/autotag/templates/rocm_changes/5.0.0.md]: This was VScode trying to be helpful and auto-updating my numbering. It should be 5.

Comment (Rmalavally) [reply tools/autotag/templates/rocm_changes/5.0.0.md]: :) Been there, done that.

**Review by saadrahim — APPROVED** [review]

File: .wordlist.txt [hunk 2]
```diff
@@ -103,7 +107,9 @@ Dependabot
 deserializers
 detections
 dev
+DevCap
```
Comment (saadrahim) [inline .wordlist.txt:110]: What is a DevCap

**Review by saadrahim — APPROVED** [review]

**Pull request merged**